src/GraphQL.SystemTextJson/DocumentWriter.cs

```diff
@@ -8,30 +8,54 @@
 
 namespace GraphQL.SystemTextJson
 {
+    /// <summary>
+    /// Serializes an ExecutionResult to a stream using the System.Text.Json library.
```

Comment:
Actually the `DocumentWriter` can write not only `ExecutionResult` (as you said in docs for `IDocumentWriter`).

Reply:
Yeah, I know.  It just seemed to be the 99% use case, and what's unique about this class is that it handles `ExecutionResult` properly.
```suggestion
    /// Serializes an ExecutionResult (or any other object) to a stream using the System.Text.Json library.
```

Reply:
OK